stratum/pipelines/stratum-tna/BUILD

```diff
@@ -0,0 +1,4 @@
+# Copyright 2021-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+# Use sh_binary?
```

Comment:
[`genrule`](https://docs.bazel.build/versions/master/be/general.html#genrule) would be nice, but I fear that `bf-p4c` will not run inside the sandbox.